tests/lua-tests/src/ParticleTest/ParticleTest.lua

```diff
@@ -29,7 +29,7 @@ end
 
 local function nextAction()
 	SceneIdx = SceneIdx + 1
-    SceneIdx = math.mod(SceneIdx, MAX_LAYER)
+    SceneIdx = math.fmod(SceneIdx, MAX_LAYER)
```

Comment:
why use `fmod`?

Reply:
luajit has been upgraded from  `2.1.0-beta2` to `2.1.0-beta3` in https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin/pull/337/files

https://github.com/elikosan/luajit-2.1/blob/master/doc/luajit.html
> Remove old Lua 5.0 compatibility defines